### PR TITLE
Changing dates to UTC

### PR DESCRIPTION
**Pull request by dhalpern**
same as previous pull request but using a different branch to follow GitHub flow

### CODE REVIEW

**Comment by deargle** [comment]
continuing discussion from https://github.com/NYUCCL/psiTurk/pull/388

I think we should standardize switch all to `datetime.now(timezone.utc)` . I wrote the "campaign" stuff, and there's tests for that. As long as the campaign unit tests pass, it's okay to change it.

**Comment by dhalpern** [comment]
I changed all the date times that I could find to UTC and it seems to have passed all tests.

File: psiturk/api/__init__.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ def post(self):
                 func=do_approve_all,
                 trigger='interval',
                 minutes=max(int(int(data['interval']) * 60), 30),
-                next_run_time=datetime.now()
+                next_run_time=datetime.datetime.now(datetime.timezone.utc)
```
Comment (deargle) [inline psiturk/api/__init__.py]: Incorrect, should not be a second datetime here

Comment (deargle) [reply psiturk/api/__init__.py]: I'm working on this. Turns out that the APScheduler needs special timezone treatment. I'll push to your PR.

**Comment by deargle** [comment]
Merged. Now the big problem, of course, is that readthedocs docs still point to the master branch, even though this is a development branch. So the docs say that utc is used, but it's actually not, for everyone on pip. Still need to reconfigure readthedocs to point to tagged releases somehow.

**Comment by deargle** [comment]
Also, reminder to self, next time we should `git squash` all these intermediary commits before merging

**Pull request merged**